angelsindustries/prototypes/overrides/components-bobs-entity-update.lua

```diff
@@ -28,177 +28,177 @@ if angelsmods.industries.components then
-              {"angels-bracket", 2},
+              {type = "item", name = "long-handed-inserter", amount = 1},
+              {type = "item", name = "circuit-green-loaded", amount = 1},
+              {type = "item", name = "angels-servo-motor-2", amount = 2},
+              {type = "item", name = "angels-bracket", amount = 2}
             }
           },
           {
             name = "filter-inserter", -- express filter inserter
             ingredients = {
               {"!!"},
-              {"red-filter-inserter", 1},
-              {"circuit-green-loaded", 2},
-              {"angels-servo-motor-2", 1},
-              {"angels-bracket", 4},
+              {type = "item", name = "red-filter-inserter", amount = 1},
+              {type = "item", name = "circuit-green-loaded", amount = 2},
+              {type = "item", name = "angels-servo-motor-2", amount = 1},
+              {type = "item", name = "angels-bracket", amount = 4}
             }
           },
           {
             name = "stack-inserter", -- express stack inserter
             ingredients = {
               {"!!"},
-              {"red-stack-inserter", 1},
-              {"circuit-orange-loaded", 2},
-              {"angels-servo-motor-3", 3},
-              {"angels-bracket", 8},
+              {type = "item", name = "red-stack-inserter", amount = 1},
+              {type = "item", name = "circuit-orange-loaded", amount = 2},
+              {type = "item", name = "angels-servo-motor-3", amount = 3},
+              {type = "item", name = "angels-bracket", amount = 8}
             }
           },
           {
             name = "stack-filter-inserter", -- express stack filter inserter
             ingredients = {
               {"!!"},
-              {"red-stack-filter-inserter", 1},
-              {"circuit-orange-loaded", 3},
-              {"angels-servo-motor-3", 2},
-              {"angels-bracket", 10},
+              {type = "item", name = "red-stack-filter-inserter", amount = 1},
+              {type = "item", name = "circuit-orange-loaded", amount = 3},
+              {type = "item", name = "angels-servo-motor-3", amount = 2},
+              {type = "item", name = "angels-bracket", amount = 10}
             }
           },
           {
             name = "turbo-inserter", -- turbo inserter
             ingredients = {
               {"!!"},
-              {"fast-inserter", 1},
-              {"circuit-orange-loaded", 1},
-              {"angels-servo-motor-3", 2},
-              {"titanium-bearing", 1},
-              {"angels-plating", 2},
+              {type = "item", name = "fast-inserter", amount = 1},
+              {type = "item", name = "circuit-orange-loaded", amount = 1},
+              {type = "item", name = "angels-servo-motor-3", amount = 2},
+              {type = "item", name = "angels-plating", amount = 2},
+              data.raw["titanium-bearing"] and {type = "item", name = "titanium-bearing", amount = 1} or nil
```

Comment:
you can leave this check for titanium bearing as our end goal is to not have these bob intermediates